.maintain/client_release.sh

```diff
@@ -15,7 +15,7 @@ RELEASE_VERSION=$(git tag --sort=committerdate | grep -E '^v[0-9]' | tail -1)
 # last two tags
 PREV_TAG=$(gh release list -L=2 | sed -n '2 p' | awk '{print $(NF-1)}')
 
-if has_client_changes "${PREV_TAG}" "${GITHUB_REF_NAME}"; then
+if has_client_changes "${PREV_TAG}" "${GITHUB_BRANCH_NAME}"; then
```

Comment:
nahh, release pipeline works on push not pull_request

Reply:
Updated